Dfe.Academies.Academisation.IService/ServiceModels/Legacy/ApplicationAggregate/LegacySchoolServiceModel.cs

```diff
@@ -21,6 +21,7 @@ public record LegacySchoolServiceModel(
 	string? SchoolConversionApproverContactEmail = null,
 
 	// conversion dates
+	bool? WantConversionToHappenOnSelectedDate = null,
```

Comment:
The point of this service model is that it is the same as the existing api endpoint, so we should not add fields in here. Does this field not map to something that's already here?

Reply:
no one has shown me / told me where the existing API endpoint is. So I have no idea whether the existing API endpoint does return this. But my guess is it doesn't. This ticket / work is part of removing UI logic and putting the Y/N in the database. I am catching up with tim hill tomo to discuss this. As james agrees with me in that the Y/N selection by users should be stored in the database. I only added it here because you told me to on another PR comment